### PR TITLE
fix(follow): preempt state machine at cancel

### PR DESCRIPTION
**Pull request by rokusottervanger**

### CODE REVIEW

File: action_server/src/action_server/actions/follow.py [hunk 1]
```diff
@@ -76,6 +76,9 @@ def _configure(self, robot, config):
 
         self._robot = robot
 
+        self._follow_sm = FollowOperator(self._robot)
+
+        self.cancel_requested = False
```
Comment (MatthijsBurgh) [inline action_server/src/action_server/actions/follow.py]: not used anywhere. correct?

Comment (rokusottervanger) [reply action_server/src/action_server/actions/follow.py]: You're totally right

Comment (MatthijsBurgh) [reply action_server/src/action_server/actions/follow.py]: What is the function of it? Any future plans?

Comment (rokusottervanger) [reply action_server/src/action_server/actions/follow.py]: No, I'll remove it. But didn't get the chance yet.

**Pull request merged**